model/src/w3snl4md.F90

```diff
@@ -1392,6 +1442,41 @@ END SUBROUTINE W3SNL4
   !!==============================================================================
   !!
   !!    -- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+  !>
+  !> @brief This routine sets up the geometric part of the Boltzmann integral.
+  !>
+  !> @details This routine sets up the geometric part of the Boltzmann integral        !
```

Comment:
I don't see the "!" at the end of other descriptions.  Should they be here too?

Reply:
This is an error.  Good catch.  I'll correct this.

Reply:
@MatthewMasarik-NOAA let me know when this is updated and then I'll do the sanity check of regtests.

Reply:
done.

Reply:
@JessicaMeixner-NOAA